scripts/sct_extract_metric.py

```diff
@@ -887,9 +944,10 @@ def save_metrics(labels_id_user, indiv_labels_ids, combined_labels_ids, indiv_la
 
     sct.printv('\tDone.')
 
-
-def check_method(method, fname_normalizing_label, normalization_method):
-    """Check the consistency of the methods asked by the user."""
+#=======================================================================================================================
```

Comment:
Actually, I want to follow the python function documentation (docstring). I will go through the scripts and convert all documentation using `""" ... """`